book/src/simple-testnet.md

```diff
@@ -1,70 +1 @@
 # Simple Local Testnet
```

Comment:
This file is just a heading, I think we should delete it?

Reply:
Thanks! `mdbook` has an annoying habit of creating a file if it's listed in `SUMMARY.md` but doesn't exist.